app/models/proofing_component.rb

```diff
@@ -1,3 +1,7 @@
 class ProofingComponent < ApplicationRecord
   belongs_to :user
+
+  def review_eligible?
+    verified_at.after?(30.days.ago)
```

Comment:
Am I reading this wrong? It's eligible for review *after* 30 days? Should it not be the opposite?

Reply:
The before? and after? methods in rails and the way this is worded is weird but what it means is the verified date is after today's date 30 days ago. I could just replace it with `verified_at > 30.days.ago` for better clarity.

Reply:
No, it's clear - I was just reading it wrong 🤪!